seqr/views/apis/staff_api_tests.py

```diff
@@ -290,11 +400,11 @@ def test_elasticsearch_status(self, mock_elasticsearch, mock_get_mapping):
 
         self.assertListEqual(response_json['diskStats'], EXPECTED_DISK_ALLOCATION)
 
-        mock_es_client.cat.allocation.assert_called_with(format="json", h="node,disk.avail,disk.used,disk.percent")
-        mock_es_client.cat.indices.assert_called_with(format="json",
-                                                      h="index,docs.count,store.size,creation.date.string")
-        mock_es_client.cat.aliases.assert_called_with(format="json", h="alias,index")
-        mock_get_mapping.assert_called_with(doc_type='variant,structural_variant')
+        mock_es_client.cat.allocation.assert_called_with(format = "json", h = "node,disk.avail,disk.used,disk.percent")
+        mock_es_client.cat.indices.assert_called_with(format = "json",
+                                                      h = "index,docs.count,store.size,creation.date.string")
+        mock_es_client.cat.aliases.assert_called_with(format = "json", h = "alias,index")
+        mock_get_mapping.assert_called_with(doc_type = 'variant,structural_variant')
```

Comment:
This does not conform with how any other code in the repo is styled. Please stick to consistent styling to keep the repo readable. Additionally, it creates work for me when I need to go through files and figure out what are real changes and what are just spacing changes. In the future, I will be asking you to do all the style cleanups before I will be reviewing PRs

Reply:
I can't find the style checking tool on the github recently. How can I open it?
Sorry for creating work for you. Would it help if I make styling changes with a separated commit?

Reply:
This isn't about the automated style checker, its about the fact that there should be nothing in the diff on github that isn't functionally relevant to the PR. If you just looked at the diff here you would see all these places where the code has different spacing that clearly don't impact the functionality

Reply:
Understood.